### ISSUE TITLE
Consider using an icon in warning text

### ISSUE THREAD

**Issue by matt-dray** [The-Strategy-Unit/nhp_inputs#641]
We have text that appears conditionally in the selection app (re asking for baseline data, not being able to run older models, etc). The text is in red, mostly to differentiate it from the other black text on the page. Consider adding a warning triangle or similar icon so that it's even more clear and accessible.

**Comment by matt-dray** [comment]
Closed by #645.
